great_expectations/datasource/fluent/interfaces.py

```diff
@@ -354,6 +354,8 @@ class Datasource(
 
     # class attrs
     asset_types: ClassVar[Sequence[Type[DataAsset]]] = []
+    # Not all Datasources require a DataConnector
```

Comment:
@Kilo59 Side comment -- I suspect that down the road all `Datasource` would do well by utilizing some kind of a `DataConnector` in order to achieve simpler, cleaner, and consistent method interfaces.  Thanks.